core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -0,0 +1,410 @@
+          allFailedTasks += exec.failedTasks;
+          allCompletedTasks += exec.completedTasks;
+          allTotalTasks += exec.totalTasks;
+          allTotalDuration += exec.totalDuration;
+          allTotalGCTime += exec.totalGCTime;
+          allTotalInputBytes += exec.totalInputBytes;
+          allTotalShuffleRead += exec.totalShuffleRead;
+          allTotalShuffleWrite += exec.totalShuffleWrite;
+          if(exec.isActive) {
+              activeExecCnt +=1;
+              activeRDDBlocks += exec.rddBlocks;
+              activeMemoryUsed += exec.memoryUsed;
+              activeMaxMemory += exec.maxMemory;
+              activeDiskUsed += exec.diskUsed;
+              activeTotalCores  += exec.totalCores;
+              activeMaxTasks += exec.maxTasks;
+              activeActiveTasks += exec.activeTasks;
+              activeFailedTasks += exec.failedTasks;
+              activeCompletedTasks += exec.completedTasks;
+              activeTotalTasks += exec.totalTasks;
+              activeTotalDuration += exec.totalDuration;
+              activeTotalGCTime += exec.totalGCTime;
+              activeTotalInputBytes += exec.totalInputBytes;
+              activeTotalShuffleRead += exec.totalShuffleRead;
+              activeTotalShuffleWrite += exec.totalShuffleWrite;
+          } else {
+              deadExecCnt +=1;
+              deadRDDBlocks += exec.rddBlocks;
+              deadMemoryUsed += exec.memoryUsed;
+              deadMaxMemory += exec.maxMemory;
+              deadDiskUsed += exec.diskUsed;
+              deadTotalCores  += exec.totalCores;
+              deadMaxTasks += exec.maxTasks;
+              deadActiveTasks += exec.activeTasks;
+              deadFailedTasks += exec.failedTasks;
+              deadCompletedTasks += exec.completedTasks;
+              deadTotalTasks += exec.totalTasks;
+              deadTotalDuration += exec.totalDuration;
+              deadTotalGCTime += exec.totalGCTime;
+              deadTotalInputBytes += exec.totalInputBytes;
+              deadTotalShuffleRead += exec.totalShuffleRead;
+              deadTotalShuffleWrite += exec.totalShuffleWrite;
+          }
+      });
+
+      var totalSummary = { "execCnt" : ( "Total(" + allExecCnt + ")"), "allRDDBlocks" : allRDDBlocks, "allMemoryUsed" : allMemoryUsed,
+          "allMaxMemory" : allMaxMemory, "allDiskUsed" : allDiskUsed, "allTotalCores" : allTotalCores, "allMaxTasks" : allMaxTasks,
+          "allActiveTasks" : allActiveTasks, "allFailedTasks" : allFailedTasks, "allCompletedTasks" :
+          allCompletedTasks, "allTotalTasks" : allTotalTasks, "allTotalDuration" : allTotalDuration,
+          "allTotalGCTime" : allTotalGCTime, "allTotalInputBytes" : allTotalInputBytes,
+          "allTotalShuffleRead" : allTotalShuffleRead, "allTotalShuffleWrite" : allTotalShuffleWrite };
+      var activeSummary = { "execCnt" : ( "Active(" + activeExecCnt + ")"), "allRDDBlocks" : activeRDDBlocks, "allMemoryUsed" : activeMemoryUsed,
```

Comment:
The labels here don't match... ie "allRDDBlocks" : activeRDDBlocks, copy and past error from totalSummary